.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,5 +17,7 @@ https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requ
 | Deprecations? | yes / no
 | Fixed ticket? | Fixes #{issue number here}.
 | How to test?  | Please indicate how to best verify that this PR is correct.
+| What are the parts of PrestaShop modified by this PR?  | Please indicate what parts of the software we need to verify to check everything is alright
```

Comment:
```suggestion
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.
```

Reply:
A smaller column will be prettier I think?
(also a detail but the number of spaces will have to be adapted in other columns)